src/components/DonationForm/DonationForm.jsx

```diff
@@ -25,12 +26,29 @@ const schema = yup.object({
 function DonationForm() {
   const {
     register,
-    // control,
+    control,
     handleSubmit,
     formState: { errors },
   } = useForm({
     resolver: yupResolver(schema),
   });
+
+  const { fields, append, remove } = useFieldArray({
```

Comment:
Consider using more descriptive names for these such as `furnitureFields`, `appendFurniture`, and `removeFurniture`.

Reply:
Note: to rename destructured variables use the following syntax
`const { val1, val2, ... } = valuesObj;`
to
`const { val1: newName1, val2: newName2, ... } = valuesObj;`